### PR TITLE
[5.4] Fix Conflict Variable Name In User Policy Class

### PR DESCRIPTION
**Pull request by ahmed-aliraqi**
When create policy for user model the methods parameters name will conflict 
So I suggest replace model variable name to $model

### CODE REVIEW

File: src/Illuminate/Foundation/Console/PolicyMakeCommand.php [hunk 1]
```diff
@@ -84,14 +84,17 @@ protected function replaceModel($stub, $model)
...
+        $stub = str_replace(
+            "use {$namespaceModel};\nuse {$namespaceModel};", "use {$namespaceModel};", $stub
+        );
 
         $model = class_basename(trim($model, '\\'));
 
         $stub = str_replace('DummyModel', $model, $stub);
 
-        $stub = str_replace('dummyModel', Str::camel($model), $stub);
+        $variableName = Str::camel($model) == 'user' : 'model' : Str::camel($model);
```
Comment (browner12) [inline src/Illuminate/Foundation/Console/PolicyMakeCommand.php]: i believe your first semicolon should be a question mark if this is a ternary.

Comment (ahmed-aliraqi) [inline src/Illuminate/Foundation/Console/PolicyMakeCommand.php]: Thanks I've already fixed it :)

**Comment by taylorotwell** [comment]
You already sent this to master branch?

**Comment by ahmed-aliraqi** [comment]
@taylorotwell I know but I mean the model variable name 😕

screenshot

**Comment by taylorotwell** [comment]
Why are you even creating a UserPolicy.

**Comment by ahmed-aliraqi** [comment]
Because they are often in the admin dashboard  who controls all the users.

**Comment by browner12** [comment]
seems like such an edge case that it's not worth the extra debt.

I would say just manually change it for this one specific case.

**Pull request closed without merging**